stylelint-config.json

```diff
@@ -28,7 +28,9 @@
 
     "property-case": "lower",
 
-    "declaration-block-no-duplicate-properties": true,
+    "declaration-block-no-duplicate-properties": [ true, {
+      "ignore": ["consecutive-duplicates"]
```

Comment:
Let's use `consecutive-duplicates-with-different-values` in case someone just accidentally duplicates a line.